common/lib/common-definitions/src/logger.ts

```diff
@@ -47,7 +47,12 @@ export interface ITelemetryGenericEvent extends ITelemetryProperties {
  * Error telemetry event.
  * Maps to category = "error"
  */
-export type ITelemetryErrorEvent = ITelemetryGenericEvent;
+export type ITelemetryErrorEvent = Omit<ITelemetryGenericEvent, "category"> & {
+    /**
+     * @deprecated - category will always be error
```

Comment:
So shouldn't this do this?
```
category: "error";
```

Reply:
keeping the change non-breaking. this will become `never` when its done being deprecated, as sendError should always set it.